api/src/test/java/jakarta/data/displaynamegeneration/ReplaceCamelCaseAndUnderscore.java

```diff
@@ -0,0 +1,88 @@
+package jakarta.data.displaynamegeneration;
```

Comment:
These files need to have copyright headers

Reply:
Done